docs/install/kata-containers-3.0-rust-runtime-installation-guide.md

```diff
@@ -49,7 +49,7 @@ Follow the [`kata-deploy`](../../tools/packaging/kata-deploy/README.md).
 
 * Download `Rustup` and install  `Rust`
     > **Notes:**
-    > Rust version 1.62.0 is needed
+    > For Rust version, please see version.yaml file's rust section.
```

Comment:
I think you could add a link for the `version.yaml`

Reply:
@houstar just please fix the typo and the rest is lgtm